scripts/variantstore/wdl/GvsJointVariantCalling.wdl

```diff
@@ -22,6 +22,7 @@ workflow GvsJointVariantCalling {
       Int extract_maxretries_override = ""
       Int extract_preemptible_override = ""
       Int extract_scatter_count = ""
+      Int load_data_batch_size = ""
```

Comment:
if you're adding a parameter to the WDL we're using for beta users, do there also need to be documentation updates?

Reply:
ah, ok — thanks for the explanation!

Reply:
only if we bubble it up to the user -- and likely we will want to hide this one and have it automatically always use the default